packages/sanity/src/core/studio/Studio.tsx

```diff
@@ -0,0 +1,30 @@
+import {ThemeColorSchemeKey} from '@sanity/ui'
+import {History} from 'history'
+import React, {ReactElement} from 'react'
+import {Config} from '../config'
+import {StudioProvider} from './StudioProvider'
+import {useWorkspace} from './workspace'
+
+/** @beta */
+export interface StudioProps {
```

Comment:
This seems hoisted from [StudioProvider.tsx](https://github.com/sanity-io/sanity/pull/3712/files#diff-37ac87a6e5318e10c4f6ff444d7fc7c4866f65714b3dc7229fd3c198303c9395)